presets/massa-station-preset.js

```diff
@@ -81,7 +85,7 @@ module.exports = {
           fontSize: "16px",
           fontWeight: "600",
           fontFamily: theme("fontFamily.Urbane"),
-          lineHeight: "19px",
+          // lineHeight: "19px",
```

Comment:
Is that intended!?

I just did the same review in Paul's MR. There's no reason to have a commented code in the codebase. If is not used, remove it.